app/components/projects/CreateProject.jsx

```diff
@@ -0,0 +1,397 @@
+import React, { createRef, useState, useContext } from 'react'
+import { Formik } from 'formik'
+import LiquidPledging from 'Embark/contracts/LiquidPledging'
+import TextField from '@material-ui/core/TextField'
+import Divider from '@material-ui/core/Divider'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Switch from '@material-ui/core/Switch'
+import Button from '@material-ui/core/Button'
+import InputAdornment from '@material-ui/core/InputAdornment'
+import CloudUpload from '@material-ui/icons/CloudUpload'
+import { withStyles } from '@material-ui/core/styles'
+import { formatForIpfs, uploadToIpfs } from '../../utils/ipfs'
+import { FundingContext } from '../../context'
+
+const { addProject } = LiquidPledging.methods
+
+
+const hoursToSeconds = hours => hours * 60 * 60
+const helperText = 'The length of time the Project has to veto when the project delegates to another delegate and they pledge those funds to a project'
+
+const styles = theme => ({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(12, [col] 1fr)',
+    gridTemplateRows: 'repeat(5, [row] auto)',
+    gridColumnGap: '1em',
+    gridRowGap: '3ch',
+    fontFamily: theme.typography.fontFamily,
+    [theme.breakpoints.up('sm')]: {
+      margin: '1.75rem 4.5rem'
+    }
+  },
+  title: {
+    display: 'grid',
+    fontSize: '2.5rem',
+    gridColumnStart: '1',
+    gridColumnEnd: '13',
+    gridRowStart: '1',
+    gridRowEnd: '6',
+    textAlign: 'center'
+  },
+  submissionRoot: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(12, [col] 1fr)',
+    gridTemplateRows: 'repeat(5, [row] auto)',
+    gridColumnGap: '1em',
+    gridColumnStart: '1',
+    gridColumnEnd: '13',
+    gridRowGap: '2ch',
+  },
+  formControl: {
+    gridColumnStart: '6'
+  },
+  formButton: {
+    gridColumnStart: '1',
+    gridColumnEnd: '13',
+    height: '50px'
+  },
+  textField: {
+    gridColumnStart: '1',
+    gridColumnEnd: '13'
+  },
+  textInput: {
+    fontSize: '2rem'
+  }
+})
+
+const isWeb = str => str.includes('http')
+const formatMedia = str => {
+  return isWeb(str) ? str : `/root/${str}`
+}
```

Comment:
Maybe better to put those functions in a utils so that we can use them in other files?